lib/src/solid/login.dart

```diff
@@ -573,3 +586,108 @@ class _SolidLoginState extends State<SolidLogin> {
     );
   }
 }
+
+class PodButton extends StatelessWidget {
+  const PodButton({
+    required this.text,
+    required this.background,
+    required this.foreground,
+    required this.tooltip,
+    required this.onPressed,
+    super.key,
+  });
+  final String text;
+  final Color background;
+  final Color foreground;
+  final String tooltip;
+  final VoidCallback onPressed;
+
+  // Define a common style for the text of the two buttons, GET POD and LOGIN.
+
+  final buttonTextStyle = const TextStyle(
+    fontSize: 12.0,
+    letterSpacing: 2.0,
+    fontWeight: FontWeight.bold,
+  );
+
+  @override
+  Widget build(BuildContext context) {
+    return Tooltip(
+      message: tooltip,
+      child: ElevatedButton(
+        // Use the onPressed passed to the widget.
+
+        onPressed: onPressed,
+        style: ElevatedButton.styleFrom(
+          backgroundColor:
+              // Use the background color passed to the widget.
+
+              background,
+          foregroundColor:
+              // Use the foreground color passed to the widget.
+
+              foreground,
+        ),
+        child: Text(
+          text,
+          style: buttonTextStyle,
+          // Style for the text inside the button can be defined here.
+        ),
+      ),
+    );
+  }
+}
+
+/// A data structure for the buttons used in the Solid Login widget.
+
+class ContinueButtonStyle {
+  const ContinueButtonStyle({
+    this.text = 'CONTINUE',
+    this.background = Colors.white,
```

Comment:
Please revert to the same colour for all. We should probably have a constant for the default button background in a theme. Avoid literals like the below colours embedded in the code.